dist/index.html

```diff
@@ -30,41 +36,47 @@ <h1 class="lg-heading">
               hesitate to contact me.
             </p>
           
-              <h2 href="#" class="connect">LET'S CONNECT</h2>
-                  <ul class="social-media">
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/twitter.svg"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/Linkedin.png"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/micro.svg"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"><img src="./img/icons/git.svg" /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
-                    </li>
-                  </ul>
-            </section>
+              <div>
+                <h2 href="#" class="connect">LET'S CONNECT</h2>
+              </div>
+                  <div>
+                    <ul class="social-media">
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/twitter.svg"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/Linkedin.png"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/micro.svg"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"><img src="./img/icons/git.svg" /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                </div>
+          </section>
 
 
-            <section class="works-card">
-                <div class="grid-container card-1">
-                    <div class="card-container">
-                      <div>
-                        <img class="card-img" src="./img/projects/Snapshoot Portfolio_1.png" alt="photo of my 1st project" />
-                      </div>
-                         <div class="text-container">
-                           <h2 class="card-title">Tonic</h3>
+          <section class="works-card">
+              <div class="grid-container grid-container-desktop">
```

Comment:
- [ ] Great job with the cards, please kindly place the text beside the image well and ensure the heading is almost aligned with the image at the top like in the original design.
yours

![Screenshot (1054)_LI](https://user-images.githubusercontent.com/65298048/236390966-949a09e4-7c8b-43a0-b559-02ed0e91c296.jpg)
figma
![Screenshot (1053)](https://user-images.githubusercontent.com/65298048/236391049-e76f150c-5adb-44bd-a875-718cdf37854f.png)